### DISCUSSION TITLE
Android docker with netsim?

### DISCUSSION THREAD

**Discussion by vChavezB** [google/bumble#577] [Q&A]
Has anyone tried exposing the netsim port over the docker-android project ?

https://github.com/budtmo/docker-android/blob/master/docker/emulator

I am trying to integrate bumble with android emulator over CI/CD

**Comment by shuowanghsu** [comment]
I have not used the docker-android project but assuming this is a wrapper for running android emulator inside docker and you have bumble running on the host outside docker - that should work as long as proper port forwarding is setup from the docker container to host. You can also utilize the "NETSIM_GRPC_PORT" environment variable to set a specific port to use.

**Reply by vChavezB** [reply]
Thanks for the hint. That worked consistently 👍
